### PR TITLE
(VoiceBot) Moving package with automatic update script to core repo

### PR DESCRIPTION
**Pull request by The-Running-Dev**

### CODE REVIEW

**Comment by The-Running-Dev** [comment]
This should be all set from my end.

File: automatic/voicebot/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿$ErrorActionPreference  = 'Stop'
+
+$downloadDir        = Get-PackageCacheLocation
+$installer          = 'VoiceBotSetup-3.0.exe'
+$arguments          = @{
+    packageName     = $env:ChocolateyPackageName
+    softwareName    = 'VoiceBot'
+    file            = Join-Path $downloadDir $installer
```
Comment (AdmiringWorm) [inline automatic/voicebot/tools/chocolateyInstall.ps1]: Remove the `file` parameter

File: automatic/voicebot/tools/chocolateyInstall.ps1 [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿$ErrorActionPreference  = 'Stop'
+
+$downloadDir        = Get-PackageCacheLocation
```
Comment (AdmiringWorm) [inline automatic/voicebot/tools/chocolateyInstall.ps1]: Remove

File: automatic/voicebot/tools/chocolateyInstall.ps1 [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+$installer          = 'VoiceBotSetup-3.0.exe'
```
Comment (AdmiringWorm) [inline automatic/voicebot/tools/chocolateyInstall.ps1]: remove

File: automatic/voicebot/update.ps1 [hunk 4]
```diff
@@ -0,0 +1,37 @@
+﻿Import-Module AU
+
+function global:au_BeforeUpdate {
```
Comment (AdmiringWorm) [inline automatic/voicebot/update.ps1]: Remove the whole function

File: automatic/voicebot/update.ps1 [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    Get-RemoteFiles -Purge
+
+    $file = Get-Item tools\*.exe | Select-Object -first 1
+    Remove-Item $file -Force -ErrorAction SilentlyContinue
+}
+
+function global:au_SearchReplace {
+    return @{
+        ".\tools\chocolateyInstall.ps1" = @{
+            "(?i)(^[$]installer\s*=\s*)('.*')" = "`$1'$([System.IO.Path]::GetFileName($Latest.Url32))'"
```
Comment (AdmiringWorm) [inline automatic/voicebot/update.ps1]: remove this replacement

File: automatic/voicebot/update.ps1 [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    $versionInfo = $downloadUrl -match $versionRegEx
+
+    if ($matches) {
+        $version = $matches[1]
+    }
+
+    return @{ Url32 = $downloadUrl; Version = $version }
+}
+
+Update
```
Comment (AdmiringWorm) [inline automatic/voicebot/update.ps1]: append `-ChecksumFor 32`

File: automatic/voicebot/voicebot.nuspec [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+    <title>VoiceBot</title>
+    <authors>Binary Fortress Software</authors>
+    <projectUrl>https://www.voicebot.net</projectUrl>
+    <licenseUrl>https://www.voicebot.net/License/</licenseUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/b586265ad8082ed125ed506419753e3f85702f86/icons/VoiceBot.png</iconUrl>
+    <docsUrl>https://www.voicebot.net/Help/</docsUrl>
+    <mailingListUrl>https://www.voicebot.net/Discussions/</mailingListUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+    ## Features
```
Comment (AdmiringWorm) [inline automatic/voicebot/voicebot.nuspec]: add a few lines of normal description before the features sub header

File: automatic/voicebot/voicebot.nuspec [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+    - Take command of your games with your voice using VoiceBot!
+    - Send commands to your games and applications
+    - Use your voice to run keyboard shortcuts, click
+    and move your mouse, and even execute complicated macros and scripts.
+    </description>
+    <copyright>Copyright © 2017 Binary Fortress Software</copyright>
+    <tags>voice control macros games trial</tags>
```
Comment (AdmiringWorm) [inline automatic/voicebot/voicebot.nuspec]: add the admin tag

File: automatic/voicebot/voicebot.nuspec [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+    <owners>chocolatey,the-running-dev</owners>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatci/voicebot</packageSourceUrl>
```
Comment (AdmiringWorm) [inline automatic/voicebot/voicebot.nuspec]: fix the typo for the automatic folder.

**Comment by The-Running-Dev** [comment]
Done!

**Comment by AdmiringWorm** [comment]
@The-Running-Dev sorry, you must have misunderstood me.
I didn't mean for you to remove the whole `SearchReplace`, only the line where you replace the value for the `$installer` variable.

Could you please update again.

**Comment by AdmiringWorm** [comment]
Another thing, have you gotten permission from the maintainer to move this package here?

**Comment by AdmiringWorm** [comment]
LGTM, once you can confirm that the maintainer have given his permission to move the package here (and it's removed from their build/update process) we can merge this one.

**Comment by AdmiringWorm** [comment]
@The-Running-Dev what's the status? Ever got a message back from the original owner?

**Comment by The-Running-Dev** [comment]
This should be all set on my end. The author is the same author as the one for LogFusiob, DisplayFusion and iTunesFusion.

**Comment by AdmiringWorm** [comment]
@The-Running-Dev I understand.
But we have yet to get any confirmation, last time we heard from you, you said:
> I had emailed the original maintainer, keithlammers, I believe he is also the guy that writes all that software. I just emailed him again to verify that he's good with it. I'll post it on here when I get it.

**Comment by The-Running-Dev** [comment]
Yes, I heard back from him and he's good with it.

**Comment by AdmiringWorm** [comment]
Seems github didn't like it when I replied from the email,
so lets try again.

Good to know, I'm not going to merge any more PR's today though (I'm just about to head for bed).
But I'll make sure to merge the open PR's from you tomorrow.

**Pull request merged**